src/System.IO.FileSystem/pkg/System.IO.FileSystem.pkgproj

```diff
@@ -2,12 +2,18 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
+    <ProjectReference Include="..\ref\4.0.2\System.IO.FileSystem.depproj">
```

Comment:
@ericstj can confirm but I think we reversion this to 4.0.1 as we haven't shipped a stable 4.0.2.